config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users, only: %i[index show]
+  resources :posts, only: %i[index show]
```

Comment:
- [ ] Since the posts belong to user, `posts` routes should be nested inside `user` routes. Nested routes make code more readable, and associations easier — nesting always gives us access to the parent id through the URL
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
- [ ] As stated by previous reviwer, Kindly use nested routes for users and posts like stated before and you can fix linters errors by run this command `rubocop -A`.